docs/pages/modules/supporting-platforms.mdx

```diff
@@ -0,0 +1,89 @@
+and the question whether to link the module in the specific CocoaPods target is moved off to the podspec.
+If you have used `"ios"` before, you can safely replace it.
+
+```diff expo-module.config.json
+{
+-   "platforms": ["ios"],
+-   "ios": {
+-     "modules": ["MyModule"]
+-   }
++   "platforms": ["apple"],
++   "apple": {
++     "modules": ["MyModule"]
++   }
+}
+```
+
+</Step>
+<Step label="2">
+
+### Update the podspec to declare support for other platforms
+
+The module's podspec needs to be updated with the supported platforms, otherwise CocoaPods would fail to install the pod on targets for the other platforms.
+As mentioned in the first step, this part of the spec is the source of truth for autolinking when the module is configured with a universal `"apple"` platform.
+
+```diff Module's podspec
+- s.platform       = :ios, '13.4'
++ s.platforms = {
++   :ios => '13.4',
++   :tvos => '13.4',
++   :osx => '10.15'
++ }
+```
+
+Note that any changes in the podspec require running `pod install` to have an effect.
+
+</Step>
+
+<Step label="3">
+
+### Set up `react-native-macos` or `react-native-tvos` in the app
+
+If you are writing a local module and your app is already set up, you can skip this step.
+Otherwise, you will need to set up your app or the example app if you are writing a standalone (non-local) module.
+
+- **For macOS**: follow the official [Install React Native for macOS](https://microsoft.github.io/react-native-windows/docs/rnm-getting-started#install-the-macos-extension) guide from `react-native-macos` documentation.
+- **For tvOS**: follow the instructions in the [react-native-tvos](https://github.com/react-native-tvos/react-native-tvos) repository.
+
+</Step>
+<Step label="4">
+
+### Review the code for using APIs not supported on these platforms
+
+Platform APIs may differ between Apple platforms. The most noticeable difference comes from relying on different UI frameworks – `UIKit` on iOS/tvOS and `AppKit` on macOS.
+
+Both `react-native-macos` and `expo-modules-core` provide some aliases and polyfills so that you can reference to `UIKit` classes on macOS target (for example `UIView` is an alias to `NSView`, `UIApplication` is an alias to `NSApplication`),
+but it's usually not enough for iOS-first libraries to support other platforms out of the box. You may need to write conditionally compiled code that uses different implementations depending on the platform.
+
+In order to do this, you can use Swift compiler directives with the `os` condition, which includes a given piece of code when our app is being built for a specific platform.
+That, combined with the `#if` and `#else` directives, essentially lets you set up platform-specific branches within the cross-platform code.
+
+```swift
+#if os(iOS)
+  // iOS implementation
+#elseif os(macOS)
+  // macOS implementation
+#elseif os(tvOS)
+  // tvOS implementation
+#endif
+```
+
+</Step>
+
+Congratulations! Your module is now ready to be used on Out-of-Tree platform.
```

Comment:
```suggestion
Your module is now ready to be used on Out-of-Tree platform.
```

Reply:
I'll remove it from here, but as far as I remember we have this in some other guides too